### PR TITLE
[DOCFIX] Update Glue User CLI

### PR DESCRIPTION
**Pull request by HelloHorizon**

### CODE REVIEW

File: docs/en/operation/User-CLI.md [hunk 1]
```diff
@@ -1265,7 +1265,12 @@ For `glue` udb type, there are some additional properties with the `-o` options:
     * `aws.catalog.id`: the aws catalog id
     * `aws.accesskey`: the aws access key id
     * `aws.secretkey`: the aws secret key
-
+    * `aws.proxy.protocol`: The protocol(HTTP/HTTPS) to use for connecting to the proxy server
```
Comment (yuzhu) [inline docs/en/operation/User-CLI.md:1268]: what about proxy url?

Comment (HelloHorizon) [reply docs/en/operation/User-CLI.md:1268]: ooops, fixed.

File: docs/en/operation/User-CLI.md [hunk 2]
```diff
@@ -1265,7 +1265,14 @@ For `glue` udb type, there are some additional properties with the `-o` options:
...
+    * `aws.proxy.host`: The proxy host the client will connect through
+    * `aws.proxy.port`: The proxy port the client will connect through
+    * `aws.proxy.username`: The proxy user name
+    * `aws.proxy.password`: The proxy password
+    * `table.column.statistics`: Enable table column statistics
```
Comment (gpang) [inline docs/en/operation/User-CLI.md:1273]: Are the values be `true` or `false`? If so, could you specify that in the description for both of these parameters?

Comment (HelloHorizon) [reply docs/en/operation/User-CLI.md]: Added.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**